test/testlib/minikube_utilities.go

```diff
@@ -1195,3 +1207,15 @@ func FindAllStatefulSets(t *testing.T, namespaceName string) NuoDBStatefulSets {
 
 	return sets
 }
+
+func RemoveOrphanNamespaces(t *testing.T) {
+	kubectlOptions := k8s.NewKubectlOptions("", "", "")
+	for _, namespace := range GetNamespaces(t) {
+		if strings.HasPrefix(namespace.Name, NAMESPACE_NAME_PREFIX) &&
+			namespace.CreationTimestamp.DeepCopy().Add(NAMESPACE_RETENTION_PERIOD).Before(time.Now()) {
```

Comment:
Do you actually need to make a copy? `Time.Add` seems to return a new `Time` struct rather than mutating the one it is invoked on.

Reply:
I'm trying to remember but I think that I checked the implementation and saw:
```
t.wall = t.wall&^nsecMask | uint64(nsec) // update nsec
t.addSec(dsec)
```

but after looking more closely in the documented example it looks like you are right.